prowler/lib/cli/parser.py

```diff
@@ -332,6 +333,12 @@ def __init_config_parser__(self):
             default=default_config_file_path,
             help="Set configuration file path",
         )
+        config_parser.add_argument(
+            "--config-fixer",
```

Comment:
```suggestion
            "--fixer-config",
```

I think it'd be better to change the order.

Reply:
You are also storing this in a `fixer_config` attribute in the provider.

Reply:
Done